src/laboratory/static/laboratory/js/laboratory.js

```diff
@@ -1,3 +1,154 @@
+function convertFormToJSON(form, prefix="") {
+  const re = new RegExp("^"+prefix);
+  return form
+    .serializeArray()
+    .reduce(function (json, { name, value }) {
+      json[name.replace(re, "")] = value;
+      return json;
+    }, {});
+}
+
+function convertToStringJson(form, prefix="", extras={}){
+    var formjson =convertFormToJSON(form, prefix=prefix);
+    formjson=Object.assign({}, formjson, extras)
+    return JSON.stringify(formjson);
+}
+
+
+function load_errors(error_list, obj){
+    ul_obj = "<ul class='errorlist shelf_form_errors d-flex justify-content-center'>";
+    error_list.forEach((item)=>{
+        ul_obj += "<li>"+item+"</li>";
+    });
+    ul_obj += "</ul>";
+    $(obj).parents('.form-group').prepend(ul_obj);
+    return ul_obj;
+}
+function load_errors(error_list, obj){
+    ul_obj = "<ul class='errorlist form_errors d-flex justify-content-center'>";
+    error_list.forEach((item)=>{
+        ul_obj += "<li>"+item+"</li>";
+    });
+    ul_obj += "</ul>"
+    $(obj).parents('.form-group').prepend(ul_obj);
+    return ul_obj;
+}
+
+function form_field_errors(target_form, form_errors, prefix){
+    var item = "";
+    for (const [key, value] of Object.entries(form_errors)) {
+        item = " #id_" +prefix+key;
+        if(target_form.find(item).length > 0){
+            load_errors(form_errors[key], item);
+        }
+    }
+}
+
+function clear_action_form(form){
```

Comment:
We should just delete the other since this one is the one that is gonna be used... you already replaced the load_errors function, so the errors no matter the method won't have the class shelf_form_errors anyway...